_usage/limitations-of-babelfish.md

```diff
@@ -180,25 +182,25 @@ features are discovered, and additional features are added to MS SQL:
 | `CREATE/ALTER/DROP PARTITION FUNCTION` | This syntax is not supported. |
 | `CREATE/ALTER/DROP PARTITION SCHEME` | This syntax is not supported. |
 | `ALTER PROCEDURE` statement | This syntax is not supported. |
-| Procedures `WITH RECOMPILE` | `WITH RECOMPILE` (when used in conjunction with the DECLARE and EXECUTE statements) is not supported. |
-| Procedure declarations with more than 100 parameters | Declarations with more than 100 parameters are not supported |
+| Procedures `WITH RECOMPILE` | `WITH RECOMPILE` (when used in conjunction with the `DECLARE` and `EXECUTE` statements) is not supported. |
+| Procedure declarations with more than 100 parameters | Declarations with more than 100 parameters are not supported. |
 | Procedure calls that includes `DEFAULT` as a parameter value | `DEFAULT` is not a supported parameter value. |
-| Procedures, externally defined | Externally defined procedures (including SQL Common Language Runtime (CLR)) are not supported |
-| Invoking a procedure whose name is in a variable | Using a variable as a procedure name is not supported |
+| Procedures, externally defined | Externally defined procedures (including SQL Common Language Runtime (CLR)) are not supported. |
+| Invoking a procedure whose name is in a variable | Using a variable as a procedure name is not supported. |
 | Procedure versioning | Procedure versioning is not supported. |
 | `CREATE/ALTER/DROP QUEUE`| Functionality related to this syntax is not supported. |
 | `READTEXT` | This syntax is not supported. |
 | Remote object access | Remote object access (including tables, views, and procedures) is not supported. |
 | `CREATE/ALTER/DROP REMOTE SERVICE BINDING` | Functionality related to this object type is not supported. |
 | `CREATE/ALTER/DROP RESOURCE POOL` | This syntax is not supported. |
-| `CREATE/ALTER/DROP EXTERNAL RESOURCE POOL` | Functionality related to this syntax is not supported |
-| `CREATE/ALTER/DROP RESOURCE GOVERNOR` | Functionality related to this syntax is not supported |
-| `RESTORE` statement | PostgreSQL snapshots of a database are dissimilar to backup files created in SQL Server. Also, there may be differences between SQL Server and PostgreSQL in the granularity of when the backup and restore occurs. |
+| `CREATE/ALTER/DROP EXTERNAL RESOURCE POOL` | Functionality related to this syntax is not supported. |
+| `CREATE/ALTER/DROP RESOURCE GOVERNOR` | Functionality related to this syntax is not supported. |
+| `RESTORE` statement | PostgreSQL snapshots of a database are different from to backup files created in SQL Server. Also, there may be differences between SQL Server and PostgreSQL in the granularity of when the backup and restore occurs. |
```

Comment:
same comment as for BACKUP: don't elaborate on PG backups

Reply:
You are right. Replaced with a similar wording as above.